
\section{Overview}

In this thesis I examined the importance of population structure and size on the accumulation of pathogen richness.
I used bats as a case study throughout due to their interesting and varied social structure \cite{kerth2008causes} and their association with a number of important, recent zoonoses \cite{leroy2005fruit, field2001natural, halpin2011pteropid, li2005bats}.
I have studied the role of these population factors using empirical, comparative approaches and simulation studies in order to both examine the specific, epidemiological mechanisms involved in a controlled and interpretable \emph{in silico} environment, while also being able to link these results back to real-world data.



\tmpsection{How I did it (Chapters overview)}



In Chapter~\ref{ch:empirical} I tested the hypothesis that bat species with more structured host populations harbour more virus species.
I tested this hypothesis with two measurements of host population structure: the number of subspecies (a novel measure and the largest data set yet used to test this hypothesis) and gene flow.
With both measures I found that, after controlling for phylogeny and study bias, a positive relationship between population structure and pathogen richness was very likely in the best model.
While the results from Chapter~\ref{ch:empirical} suggest that there is a relationship between population structure and pathogen richness, comparative studies like these cannot identify which specific mechanisms maintain high pathogen richness.
I therefore used simulations to test whether population structure (specifically network topology and dispersal rate) could allow a newly evolved pathogen to invade and persist in the presence of strong competition from an established, endemic pathogen (Chapter~\ref{ch:sims1}).
However, I found the opposite relationship to Chapter~\ref{ch:empirical}; I found that decreasing host population structure increased the rate of pathogen invasion.
In Chapter~\ref{ch:sims2} I used the same model as Chapter~\ref{ch:sims1} to test whether host population size or density more strongly promoted pathogen invasion and establishment and whether a pathogen invaded more easily into a population comprising many small colonies or fewer big colonies.
I found that population size had a much stronger effect than density on the probability of pathogen invasion and that colony size had a much stronger effect than the number of colonies.
Theory \cite{may1979population, anderson1979population}, previous literature \cite{kamiya2014determines, nunn2003comparative, morand1998density} and Chapters \ref{ch:sims1} and \ref{ch:sims2} suggested that population size (either local group size or global population size) strongly influences the dynamics of disease and pathogen richness.
However, this variable was not included in the empirical study in Chapter~\ref{ch:empirical} as there are very few estimates of population size for bats and colony counts are time consuming and costly \cite{kloepper2016estimating}.
However data from acoustic camera trap surveys are increasingly available \cite{jones2011indicator}.
To make the estimation of population sizes easier, for bats and other acoustic detectable animals, I developed a general method for estimating population size and density from acoustic detectors or camera traps (Chapter~\ref{ch:grem}).


\section{Comparison to the literature}

There is a common assumption that factors that increase $R_0$ should increase pathogen diversity \cite{nunn2003comparative, morand2000wormy}.
However, my results imply a more nuanced relationship. 
In Chapter~\ref{ch:sims1} I found that reduced global population structure promoted the invasion of new pathogen species.
In Chapter~\ref{ch:sims2}, I found that while global host population density --- which affected population structure --- had an effect on invasion rate, group size had a much stronger effect.
In contrast, in Chapter \ref{ch:empirical}, I found the opposite relationship; that in wild bat populations, increased host population structure promotes pathogen richness.
One interpretation of this is that there are two distinct phases to pathogen competition.
When a new pathogen first enters a population, many contacts (i.e. a highly connected population) allows the pathogen to spread and avoid stochastic extinction.
However, after this initial spread, host population structure may enable the pathogen to persist for longer.
There is a positive empirical relationships between pathogen richness and host population structure (Chapter~\ref{ch:empirical}, \textcite{turmelle2009correlates}, \textcite{maganga2014bat}) and population structure and range size \cite{kamiya2014determines, nunn2003comparative} while in contrast the evidence for a positive relationship between pathogen richness and  group size is equivocal \cite{rifkin2012animals, ezenwa2006host}.
This contrast suggests that population structure in wild species affects the latter stages of pathogen dynamics, enabling pathogen persistence, more than it affects the initial invasion of pathogens.
Little research has so far been conducted contrasting these different processes and examining which mechanisms could promote high pathogen richness at each scale.

%Much research in multipathogen systems has been conducted over the short time scales of a single epidemic \cite{van2014domination, poletto2013host, poletto2015characterising, funk2010interacting}.
%While this time scale has important human health consequences, when examining the slow process of the accumulation of pathogen species, a longer term view needs to be examined.
%Interestingly, my results, along with previously published studies show quite strong differences between these timescales. 
%Competing epidemics are strongly affected by population structure with structure promoting coexistence of pathogens and allowing less competitive pathogens to persist \cite{poletto2013host, poletto2015characterising}.
%In contrast, in the longer time scales studied here in the empirical study, I have found that population structure does not seem to allow an invading pathogen to escape competition (Chapters \ref{ch:sims1} and \ref{ch:sims2}).
%This can be understood by considering that at very long time scales, any population is well mixed unless there is complete separation of subpopulations.





\section{Other mechanisms controlling pathogen richness}

Colony size has been found to have both a negative relationship \cite{gay2014parasite} and no relationship \cite{turmelle2009correlates} with parasite richness in previous comparative studies using relatively small data sets.
However, in Chapter \ref{ch:sims2} I found that colony size is particularly important for promoting pathogen richness.
I did not include colony size in my comparative analysis (Chapter \ref{ch:empirical}) for three reasons.
Firstly, the focus of the chapter was broad-scale population structure.
Secondly, there was a lack of previously published, strong evidence of a relationship between colony size and pathogen richness.\cite{turmelle2009correlates}.
Finally, there is a considerable lack of data on colony size and I was aiming for a large sample size.
However, given the results of Chapter~\ref{ch:sims2}, filling these data gaps would be a useful avenue for further research.
In particularly, testing the relative effects of population density and colony size would be a useful test of the model used in Chapter~\ref{ch:sims2}.

In this thesis I have only examined one mechanism by which population-level factors may affect pathogen richness.
I have only examined the ability of a newly evolved pathogen (i.e. a pathogen identical to an endemic pathogen and in the presence of strong competition) to invade and persist.
However, there are a number of other mechanisms that could equally strongly affect pathogen richness in the wild.
Closely related to the mechanism studied here is the case of pathogens invading from other host species.
These pathogens are likely to have different epidemiological parameters (transmission rate, virulence, recovery rate) to the endemic pathogen as modelled in \textcite{may1994superinfection} for example.
Furthermore, the competition between pathogens is expected to be less strong.
Alternatively, host population traits could affect the rate of pathogen extinction.
Once a number of pathogens are established in a population, there is still likely to be occasional extinctions, especially in the presence of inter-pathogen competition \cite{kapusinszky2015local}. %todo ref
A number of population factors could affect this rate.
It is expected that large populations will experience slower rates of pathogen loss as stochastic extinction will be more rare.
Furthermore, populations with strongly varying disease prevalences are likely to have higher rates of pathogen extinction.
Higher rates of pathogen extinction are likely to occur in populations where epidemic cycles are common \cite{altizer2006sesonality}.
After a large epidemic, the number of susceptible individuals in the population will be low due to immunity, host death or low birth rates induced by infection \cite{scott1987population, hethcote1994thousand}.
While the number of susceptibles is low, stochastic extinction is more likely. %ref 
This effect will be exacerbated in the case where an epidemic cycle is synchronous across the whole population as is the case in unstructured populations \cite{duke2011strong, mckenzie2001seasonality}.
Structured populations with asynchronous epidemic cycles may experience local pathogen extinction but rarely global extinction; this pattern of local extinction and recolonisation has been well studied in the ecological literature \cite{grenfell1995seasonality, levin1974dispersion, hanski1998metapopulation}, but less so in the epidemiological literature. %todo read http://link.springer.com/article/10.1007/s10980-008-9245-4


\section{Predictive modelling}

I have found evidence, both empirical and theoretical, that demographic parameters can influence pathogen richness.
However it seems likely that this effect alone is not strong enough to be a useful predictor of viral richness with respect to surveillance for zoonotic diseases.
While there is potential for population structure and colony size to be useful variables when combined with other variables in a predictive framework, the biases in all pathogen richness datasets makes these approaches difficult.
However, as more unbiased data is collected --- as in \textcite{anthony2013strategy, anthony2015non} --- or using much larger pathogen data sets --- such as \textcite{wardeh2015database} --- predictive models may become a more viable tool.
Furthermore, the method provided in Chapter~\ref{ch:grem} makes the collection of population size data more feasible over broad taxonomic, spatial or temporal scales, further increasing the potential of predictive models.

One approach to dealing with the biases in the data would be to use small, unbiased pathogen richness datasets as test data while using the large, biased data sets as training data.
This use of data that predicts a related but different measure is known as domain adaptation \cite{daume2006domain, daume2009frustratingly}.
As pathogen richness will be measured on different scales in these datasets, steps will have to be taken to make the information in the larger data sets applicable to the smaller datasets.
It might be possible to predict a species to pathogen richness quantile or broad pathogen richness classes (i.e.\ ``high pathogen richness'' and ``low pathogen richness'').
This may be informative enough for advising policy and prioritising species for zoonotic surveillance.
Alternatively, a subset of the small unbiased data sets could be used to create a simple model to rescale predictions from the model trained on the large dataset so that it is applicable to the small dataset \cite{daume2009frustratingly}.

While predictive models should be built, forecasting changes in pathogen richness and zoonotic risk will be difficult.
However, the mechanistic understanding obtained by the theoretical chapters here can suggest how pathogen richness may respond to global change and other population stressors.
Firstly, when global change acts to reduce group size \cite{lehmann2010apes, zunino2007habitat, manor2003impact, atwood2006influence} pathogen richness is expected to decrease.
Conversely, in species where group size is increasing \cite{lehmann2010apes}, pathogen richness is expected to increase.
Species suffering range contractions \cite{thomas2004extinction} and decreases in population size \cite{craigie2010large} are expected to experience smaller changes in pathogen richness despite these being the more commonly studied effects of global change.
This suggests that further research should study in more detail the effects of climate change on social group size.

%Furthermore, I have shown that while population factors such as population size, density and range size are directly linked, they have very different effects on pathogen richness.
%Therefore future studies should be careful to acknowledge these relationships and, where possible, compare multiple demographic measurements to further test which factors are in fact causally affecting pathogen richness.



\section{Bat social structure}

It is important to note that I have ignored much of the social complexity found in bats.
Information on these other social behaviours was not explicitly included in the empirical study in Chapter \ref{ch:empirical}.
Furthermore, in Chapters \ref{ch:sims1} and \ref{ch:sims2} I have modelled bat populations as a metapopulation where the only social structure is the grouping of individuals into subpopulations.
There is dispersal between these subpopulations but otherwise they are static.
Firstly, I have not modelled the creation of new colonies, or the disbanding of colonies \cite{metheny2008genetic}.
Especially in the face of habitat destruction, it is likely that the number of colonies of a species will be decreasing.
Furthermore, in some species, colonies are likely to be more fluid, with groups joining and splitting \cite{kerth2012causes, august2014sympatric}.
Secondly, there are a number of behaviours common in bats, particularly in temperate regions, that has been excluded from these models.
For example, many species have different types of colonies --- maternity colonies, mating colonies and hibernation colonies \cite{kerth2008causes}.
Epidemiological dynamics are likely to be altered by the physiological differences in bats while in these different colony types  but also due to their role in population structure \cite{george2011host, langwig2015host, blehert2012fungal, webber2016social}.
The extent to which the individuals move together when switching between these colony types is largely unknown \cite{kurta2002philopatry, baerwald2016migratory} but if there is a large degree of mixing during the transition between colony types, then there will be considerably less population structure overall.
Similarly, swarming behaviour --- the coming together of many bats from different colonies --- is likely to decrease epidemiological population structure \cite{kerth2012causes}.


Furthermore, many bat species, both temperate and tropical, are migratory \cite{fleming2003ecology, krauel2013recent, popa2009bats, hutterer2005bat}.
Again, it is largely unknown whether colonies travel together during migration \cite{baerwald2016migratory}. 
It is therefore also unknown whether colony structure is similar before and after migration \cite{carter2013cooperation} though \textcite{kurta2002philopatry} find that individuals do not migrate together.
There is also little data on whether parameters, such as intercolony dispersal rate, are constant before and after migration.
Even if colonies remain fairly constant during migration, the spatial relationships may be different; colonies that were far apart in one area could subsequently be near neighbours after migration.
However, migratory status has been included in previous comparative analyses and not been found to be a strong predictor of pathogen richness \cite{turmelle2009correlates, maganga2014bat}. 

Another potentially important factor that has been ignored here is roost sharing by different bat species \cite{maganga2014bat, lopez2014seroprevalence, serra2002european, pons2014insights, deThoisy2016bioecological}.
If the species are very similar in most epidemiological factors, this could potentially be sensibly modelled by ignoring species identity and treating the whole population as one.
However, it is more likely that there will be fewer close contact events between individuals of different species even if they roost share.
It is also likely that species will have different dispersal patterns between colonies.
Therefore, more complex models such as overlay network models might be needed in order to effectively model these populations \cite{funk2010interacting, marceau2011modeling}.
Roost sharing and the amount of sympatry has been included in comparative studies of bat pathogen richness \cite{maganga2014bat} but was not found to correlate with pathogen richness.

Finally, birth and deaths have been modelled here as occurring randomly through time but many bat species have very tightly controlled birth pulses \cite{dietrich2015leptospira, george2011host, porter2001birth, greiner2011predictable}.
This has important epidemiological consequences; there will be a pulse of susceptible individuals each year with very few new susceptibles during the rest of the year \cite{dietrich2015leptospira}.
Models of these population dynamics have found that birth pulses can drive pathogen extinction \cite{peel2014effect}.
\textcite{hayman2015biannual} found that certain Filoviruses were less likely to persist in bat species with an annual birth pulse than a biannual birth pulse.
In other mammals, birth pulses have also been shown to reduce synchrony of dynamics \cite{duke2011strong}.

Overall, there is much complexity that could be added to epidemiological models of bats.
However, there is little data for many species which makes parameterisation difficult.
Furthermore, as these factors differ between species, trying to make general models that apply across the order is difficult.
Further work should include specific, detailed models of well studied species and further examination of how important these various factors might be.


%\begin{itemize}
%\item \sout{Collect data for colony size and test importance against structure.}
%\item Limitations of poor data, escpecially for gen flow.
%\item \sout{bias of viral data. Perhaps fixed by non-biased sampling or larger datasets.}
%\item \sou{Examine other mechanisms for richness}
%\item \sout{Examine multi host species more carefully}
%\item \sout{Field test gREM}
%\item \sout{Use gREM to collect density estimates }
%\end{itemize}


\section{Conclusions}

%\begin{itemize}
%\item Population structure does influence pathogen richness but the mechanisms are still unclear.
%\item Local dynamics (local density) are most important for pathogen invasions not broad scale structure.
%\item Data on density should be collected using the gREM.
%\end{itemize}

Overall my studies suggest that population size and structure have an important role in controlling pathogen richness. 
However, my two studies on this topic give contradictory results and so the exact mechanisms by which these effects occur are still not clear.
I have found that population size and colony size is particularly important for controlling pathogen richness in the case of closely related, strongly competing pathogens.
I have also provided a tool to facilitate the estimation of population sizes in echolocating bats and other mammals.



